If-Statements/sound_levels/sound_levels.rs

```diff
@@ -0,0 +1,46 @@
+use std::io;
+
+fn main() {
+    let mut sound_level = String::new();
+    
+    println!("Enter a sound level in dB");
+    io::stdin().read_line(&mut sound_level)
+        .expect("Failed to read line.");
+
+    let sound_level: u32 = match sound_level.trim().parse() {
+        Ok(num) => num,
+        Err(_) => 0
+    };
+
+    // The == operator checks if both values are the same
+    if sound_level == 130 {
+        println!("This is equivalent to a jackhammer.");
+    } else if sound_level == 106 {
+        println!("This is equivalent to a lawnmower.");
+    } else if sound_level == 70 {
+        println!("This is equivalent to an alarm clock.");
+    } else if sound_level == 40 {
+        println!("This is equivalent to a quiet room.");
+    }
+
+    // The following operators can also be used for conditional programming in Rust:
+    // < - less than 
+    // > - greater than
+    // <= - less than or equal to
+    // >= - greater than or equal to
+    // && - and 
+    // || - or
+    // ! - not
+    // != - not equal to 
```

Comment:
Handy operator list there, maybe we could add these to a Wiki or something for this repo ...

Reply:
Maybe a link to the ["Operators and Symbols"](https://doc.rust-lang.org/book/second-edition/appendix-02-operators.html) appendix of "The Book" would be more informative and complete :sweat_smile:. I probably left out like 10 relating to if statements :D